src/util/misc/projectStroke/index.ts

```diff
@@ -0,0 +1,53 @@
+import { IPoint } from '../../../point.class';
+import { StrokeLineCapProjections } from './StrokeLineCapProjections';
+import { StrokeLineJoinProjections } from './StrokeLineJoinProjections';
+import { TProjection, TProjectStrokeOnPointsOptions } from './types';
+
+/**
+ *
+ * Used to calculate object's bounding box
+ *
+ * @see https://github.com/fabricjs/fabric.js/pull/8344
+ *
+ */
+export const projectStrokeOnPoints = (
+  points: IPoint[],
+  options: TProjectStrokeOnPointsOptions,
+  openPath = false
+): TProjection[] => {
+  const projections: TProjection[] = [];
+
+  if (points.length <= 1) {
+    return projections;
```

Comment:
Here is the 1 point edge case that we need to consider returning a round projection

Reply:
solved

Reply:
needs validation